Kudu.Core/Deployment/WellKnownEnvironmentVariables.cs

```diff
@@ -38,5 +38,9 @@ internal static class WellKnownEnvironmentVariables
 
         public const string GoWebConfigTemplate = "GO_WEB_CONFIG_TEMPLATE";
         public const string SelectLatestVersionCommandKey = "SELECT_LATEST_VERSION_CMD";
+
+        public const string GypMsvsVersion = "GYP_MSVS_VERSION";
+        public const string VCTargetsPath = "VCTargetsPath";
```

Comment:
These are actually consumed by MSBuild.exe so they have to match what MSBuild is expecting.